src/utils.js

```diff
@@ -454,3 +485,29 @@ exp.detectBirthdaylessUsers = async robot => {
     }
   }
 }
+
+/**
+ * Send count of users which are agreed for begging money
```

Comment:
Inform everybody in the birthday channel about a number of people who agreed to pitch in.